indico/MaKaC/services/implementation/registration.py

```diff
@@ -101,8 +102,8 @@ def _getAnswer(self):
                    "date": format_date(self._conf.getAdjustedStartDate()),
                    "server": {"baseUrl": baseURL,
                               "consumerKey": checkin_app.client_id,
-                              "consumerSecret": checkin_app.client_secret,
-                              }
+                              "auth_url": url_for('oauth.oauth_authorize'),
+                              "token_url": url_for('oauth.oauth_token')}
                    }
```

Comment:
Don't you want `_external=True` for those URLs so you get absolute ones?

Also, the `}` should go in the previous line...